src/profile-logic/gecko-profile-versioning.js

```diff
@@ -331,5 +331,78 @@ const _upgraders = {
     }
     convertToVersionTenRecursive(profile);
   },
+  [11]: profile => {
+    const categories = [
+      {
+        name: 'Idle',
+        color: 'transparent',
+      },
+      {
+        name: 'Other',
+        color: 'grey',
+      },
+      {
+        name: 'JavaScript',
+        color: 'yellow',
+      },
+      {
+        name: 'Layout',
+        color: 'purple',
+      },
+      {
+        name: 'Graphics',
+        color: 'green',
+      },
+      {
+        name: 'DOM',
+        color: 'blue',
+      },
+      {
+        name: 'GC / CC',
+        color: 'orange',
+      },
+      {
+        name: 'Network',
+        color: 'lightblue',
+      },
+      {
+        name: 'HTML Parsing',
+        color: 'brown',
+      },
+    ];
+    const oldCategoryToNewCategory = {
```

Comment:
nit: can you please put links (with fixed versions of course) to the Firefox code that defines old and new categories ?